src/main/java/it/gov/pagopa/pu/pagopapayments/exception/SynchronousPaymentException.java

```diff
@@ -0,0 +1,13 @@
+package it.gov.pagopa.pu.pagopapayments.exception;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+
+@EqualsAndHashCode(callSuper = true)
+@RequiredArgsConstructor
+@Data
+public class SynchronousPaymentException extends ApplicationException {
+  private final String errorCode;
```

Comment:
could be more usefull to ask directly the enum here?
```suggestion
  private final PagoPaNodeFaults error;
```

Reply:
done